docs/website/src/landing/features/features.less

```diff
@@ -1,15 +1,15 @@
 @import (reference) "../../common/variables.less";
 
 @features-marker: escape('
-  <svg width="19" height="46" viewBox="0 0 19 46" fill="none" xmlns="http://www.w3.org/2000/svg">
-    <line x1="17.0146" y1="1.28171" x2="1.28171" y2="44.5076" stroke="@{landing-blue}" stroke-width="2" stroke-linecap="round"/>
+  <svg width="19" height="100" viewBox="0 0 19 100" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
width and height are unnecessary attributes because they have already been presented in the viewBox attribute